gratipay/billing/exchanges.py

```diff
@@ -82,6 +83,7 @@ def create_card_hold(db, participant, amount):
             'options': { 'submit_for_settlement': False },
             'custom_fields': {'participant_id': participant.id}
         })
+        ref = result.transaction.id
```

Comment:
After a ridiculous long time researching it...yes we can build a URL to the transaction using `transaction.id`
I believe it would look something like
`https://dashboard.braintreegateway.com/merchants/gratipay/transactions/1vt8z5a3`

where `1vt8z5a3` is the `transaction.id`